### PR TITLE
feat(blocks manager): Create a bare-bones version of Blocks Manager

### PR DESCRIPTION
**Pull request by anler**
This PR includes:

- [x] Creating the scaffold for the `BlocksManager` actor without adding any particular feature to it (no handlers).
- [x] Starting up `BlocksManager` from `core/actors/node.rs`
- [x] Create basic documentation page for `BlocksManager`

### CODE REVIEW

File: docs/architecture/managers/blocks-manager.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Blocks Manager
+
+The __blocks manager__ is the actor in charge of managing the blocks of Witnet blockchain that resides in the node...
```
Comment (aesedepece) [inline docs/architecture/managers/blocks-manager.md]: This is WIP, right? :grin:

Comment (aesedepece) [reply docs/architecture/managers/blocks-manager.md]: As discussed, we should put here some basic description of the responsibilities of this manager. Issue #123 may already have what we need.

**Comment by anler** [comment]
Yes, I pulled the string of issues and found the info in the epic already :man_facepalming: fixing it as I speak!

**Comment by mmartinbar** [comment]
This PR is superseded by #158

**Pull request closed without merging**